scripts/update_robot.py

```diff
@@ -183,7 +184,55 @@ def on_shutdown():
     return nl.rc
 
 
+def rethink_ssh_update_version():
```

Comment:
This should probably be renamed `rethink_workstation_update_version()` since it returns `False` when the update needs to take place over SSH and `True` when this script can proceed.